src/components/domain/ShoppingItem/styled.ts

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled';
+import type { ShoppingItemProps } from './types';
+import { COLOR } from '@utils/constants/colors';
+
+const StyledContainer = styled.div<ShoppingItemProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  gap: 5px;
+  width: 800px;
+  height: 120px;
+  background-color: ${COLOR.BASIC_WHITE};
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  border: 2px solid ${COLOR.BASIC_WHITE};
+  border-radius: 20px;
+  margin: 20px 0px;
+`;
+
+const StyledImage = styled.div<ShoppingItemProps>`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+`;
+
+const StyledTitle = styled.div<ShoppingItemProps>`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 380px;
+`;
+
+const StyledVerticalLine = styled.div<ShoppingItemProps>`
+  border-left: 1px solid ${COLOR.LIGHT_GRAY};
+  height: 100px;
+`;
+
+const StyledInfo = styled.div<ShoppingItemProps>`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 220px;
```

Comment:
```suggestion
const StyledContainer = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: row;
  gap: 5px;
  width: 800px;
  height: 120px;
  background-color: ${COLOR.BASIC_WHITE};
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
  border: 2px solid ${COLOR.BASIC_WHITE};
  border-radius: 20px;
  margin: 20px 0px;
`;

const StyledImage = styled.div`
  display: inline-flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  width: 200px;
`;

const StyledTitle = styled.div`
  display: inline-flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  width: 380px;
`;

const StyledVerticalLine = styled.div`
  border-left: 1px solid ${COLOR.LIGHT_GRAY};
  height: 100px;
`;

const StyledInfo = styled.div`
  display: inline-flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  width: 220px;
```

해당 Generic은 여기서는 특별히 필요없을것같아요!

Reply:
지금 보니 그렇네요! 타입 프롭스가 사용되지도 않는데 왜 써놨을까요 허허...

Reply:
수정 완료했습니다-!